paasta_tools/tron/tron_command_context.py

```diff
@@ -75,58 +51,11 @@ def __ne__(self, other):
         return not self == other
 
 
-class JobContext:
```

Comment:
I was surprised we had this in here, but I can't figure out how we were/are using this. @chlgit or @qui ?

Reply:
I don't see it in any private validation hooks

Reply:
It was added in this commit but never referenced: 5231c6e3c420265b2027c715de1aa22691e5a786

Reply:
I think we copied all of https://github.com/Yelp/Tron/blob/master/tron/command_context.py. This was to support Tron-like command context variables in Chronos, before Tron on PaaSTA was a thing.